frontend/src/app/AboutDialog.tsx

```diff
@@ -136,7 +136,11 @@ const AboutDialog: React.FC<AboutDialogProps> = ({ onClose }) => {
           >
             <h4>Installed components</h4>
             {groupedComponents.length > 0 ? (
-              <Table aria-label="Component Releases Table" data-testid="component-releases-table">
+              <Table
```

Comment:
This table doesn't use our base table? lol -- this seems not great (unrelated to this PR)

Reply:
You're right - great catch there @andrewballantyne! I can create a JIRA and link it here as a follow up to replace this with `TableBase`.

Reply:
Added a TODO comment.